### PR TITLE
Release Google.Api.Gax version 4.0.0-alpha02

### PR DESCRIPTION
**Pull request by jskeet**
This is *not* intended for public use; it's for the .NET client
libraries team to consume while working on the client libraries
themselves and the generator.

### CODE REVIEW

**Comment by jskeet** [comment]
(Will wait until #561 is merged before merging this.)

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**
